src/cephadm/tox.ini

```diff
@@ -54,8 +54,10 @@ commands =
 
 [testenv:flake8]
 basepython = python3
+allowlist_externals = bash
 deps =
     flake8
     flake8-quotes
 commands =
     flake8 --config=tox.ini {posargs:cephadm}
+    bash -c "test $(grep 'docker.io' cephadm | wc -l) == 10"
```

Comment:
Was this before or after the snmp backport? Don't want to need a repeat of https://github.com/ceph/ceph/pull/44583